src/collector.rs

```diff
@@ -58,10 +58,17 @@ impl Collector for SoftwareVersion {
     }
 
     fn collect(&mut self, crate_info: &CrateInfo) -> Result<ReportEntry> {
+        #[cfg(feature = "git_hash")]
+        let git_hash_suffix = format!(" {}", crate_info.git_hash);
```

Comment:
Can we maybe add the git hash in parenthesis?
"bugreport 0.3.0 (a984113)"

Reply:
I actually played around with that locally, but arbitrarily ended up having no parenthesis. I’ll happily add them.

Reply:
Fixed in 4c2ee87 that I just pushed to this MR.